### PR TITLE
fix census checks on NewProcessEvent

### PR DESCRIPTION
**Pull request by jordipainan**
WIP: waiting for censusOriginProperties @p4u

### CODE REVIEW

File: chain/contractWrappers.go [hunk 1]
```diff
@@ -114,16 +114,13 @@ func (ph *VotingHandle) NewProcessTxArgs(ctx context.Context, pid [types.Process
...
-	if cOrigin > types.ProcessesContractMaxCensusOrigins {
-		return nil, fmt.Errorf("invalid census origin: %d", processMeta.ModeEnvelopeTypeCensusOrigin[2])
-	}
-	processData.CensusOrigin = models.CensusOrigin(cOrigin)
-	evmCensus := processData.CensusOrigin == models.CensusOrigin_OFF_CHAIN_TREE
-	// census mkuri, only for off chain censuses
-	if evmCensus {
-		processMeta.MetadataCensusRootCensusUri[2] = util.TrimHex(processMeta.MetadataCensusRootCensusUri[2])
-		processData.CensusURI = &processMeta.MetadataCensusRootCensusUri[2]
+	if processData.CensusOrigin = models.CensusOrigin(cOrigin); processData.CensusOrigin == 0 {
```
Comment (p4u) [inline chain/contractWrappers.go]: Are u sure of this? If a ENUM does not exist on Proto the result is `0` ?

File: types/consts.go [hunk 2]
```diff
@@ -93,9 +93,6 @@ const (
 	TxAddProcessKeys    = "addProcessKeys"
 	TxRevealProcessKeys = "revealProcessKeys"
 
-	// ProcessesContractMaxCensusOrigins represents the max value that a uint8 can have
-	// with the current smart contract bitmask describing the supported census origins
-	ProcessesContractMaxCensusOrigins = 7
 	// ProcessesContractMaxProcessMode represents the max value that a uint8 can have
 	// with the current smart contract bitmask describing the supported process mode
 	ProcessesContractMaxProcessMode = 15
```
Comment (p4u) [inline types/consts.go:98]: Is this var still used? If not, please remove it.

File: types/vochain.go [hunk 3]
```diff
@@ -128,3 +128,20 @@ type ScrutinizerOnProcessData struct {
...
+type CensusProperties struct {
+	Name            string
+	NeedsDownload   bool
+	NeedsIndexSlot  bool
+	NeedsURI        bool
+	WeightedSupport bool
+}
+
+var CensusOrigins = map[models.CensusOrigin]CensusProperties{
+	models.CensusOrigin_OFF_CHAIN_TREE:          {Name: "offchain tree", NeedsDownload: true, NeedsURI: true},
```
Comment (jordipainan) [inline types/vochain.go:143]: Suggest to change the `Name` hard-coded strings to constants

Comment (p4u) [reply types/vochain.go:143]: Mmm, not sure... To me its clear to declare all properties (including human friendly name) on the same place.

**Comment by jordipainan** [comment]
LGTM

**Pull request merged**